governator-core/src/main/java/com/netflix/governator/event/ApplicationEvent.java

```diff
@@ -0,0 +1,10 @@
+package com.netflix.governator.event;
+
+/**
+ * Marker interface for application events to be published
+ * to subscribed consumers.
+ */
+public interface ApplicationEvent {
```

Comment:
I would vote for keeping it. Having marker interfaces is a common pattern + it helps in discovering event data types by checking the inheritance tree.